.github/workflows/python.yml

```diff
@@ -186,5 +186,3 @@ jobs:
       - name: Publish 📦 to PyPI
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         uses: pypa/gh-action-pypi-publish@release/v1
-        with:
-          password: ${{ secrets.pypi_password }}
```

Comment:
if the password is not used anymore, then how is it publishing to pypi?

Reply:
https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/

Reply:
1) why does testpy have the password then still
2) where do we give the permission for ci to generate an identity token

Reply:
1. Because test pypi uses a token that is for my entire account so I don't need to bother with per-project configs
2. I seem to have not copied that part